### ISSUE TITLE
XCode 10 GM and Swift 4.2 fails to build

### ISSUE THREAD

**Issue by jlaws** [kaishin/Gifu#133]
```
/Users/joelaws/Workspace/kratos/Pods/Gifu/Source/Classes/GIFAnimatable.swift:16:20: 'UIViewContentMode' has been renamed to 'UIView.ContentMode'
/Users/joelaws/Workspace/kratos/Pods/UIKit.UIViewContentMode:2:18: 'UIViewContentMode' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/Gifu/Source/Classes/Animator.swift:133:75: 'UIViewContentMode' has been renamed to 'UIView.ContentMode'
/Users/joelaws/Workspace/kratos/Pods/UIKit.UIViewContentMode:2:18: 'UIViewContentMode' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/Gifu/Source/Classes/Animator.swift:149:72: 'UIViewContentMode' has been renamed to 'UIView.ContentMode'
/Users/joelaws/Workspace/kratos/Pods/UIKit.UIViewContentMode:2:18: 'UIViewContentMode' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/Gifu/Source/Classes/Animator.swift:73:87: 'UIViewContentMode' has been renamed to 'UIView.ContentMode'
/Users/joelaws/Workspace/kratos/Pods/UIKit.UIViewContentMode:2:18: 'UIViewContentMode' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/Gifu/Source/Classes/Animator.swift:92:84: 'UIViewContentMode' has been renamed to 'UIView.ContentMode'
/Users/joelaws/Workspace/kratos/Pods/UIKit.UIViewContentMode:2:18: 'UIViewContentMode' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/Gifu/Source/Classes/Animator.swift:105:41: 'RunLoopMode' has been renamed to 'RunLoop.Mode'
/Users/joelaws/Workspace/kratos/Pods/Foundation.RunLoopMode:2:18: 'RunLoopMode' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/Gifu/Source/Classes/FrameStore.swift:29:20: 'UIViewContentMode' has been renamed to 'UIView.ContentMode'
/Users/joelaws/Workspace/kratos/Pods/UIKit.UIViewContentMode:2:18: 'UIViewContentMode' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/Gifu/Source/Classes/FrameStore.swift:86:47: 'UIViewContentMode' has been renamed to 'UIView.ContentMode'
/Users/joelaws/Workspace/kratos/Pods/UIKit.UIViewContentMode:2:18: 'UIViewContentMode' was obsoleted in Swift 4.2
```

**Comment by kaishin** [comment]
Fixed in 3.2.0!

**Comment by jlaws** [comment]
Works great, thank you!
